frontend/packages/console-shared/src/components/highlight/Highlight.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import Portal from '../popper/Portal';
+import './Highlight.scss';
+
+type HighlightProps = {
+  selector: string;
+};
+
+const Highlight: React.FC<HighlightProps> = ({ selector }) => {
+  const element = document.querySelector(selector);
+  const { height, width, top, left } = element.getBoundingClientRect();
+  const style: React.CSSProperties = {
+    top,
+    left,
+    height,
+    width,
+  };
+  return (
+    <Portal>
```

Comment:
The implementation is exactly same as the Spotlight component from guided tour. Instead of creating a new one you should just move that one to console-shared and add some customization for this use case.

Reply:
updated with customized spotlight component